Packs/Humio/Integrations/Humio/Humio.py

```diff
@@ -234,86 +234,109 @@ def humio_delete_alert(client, args, headers):
     else:
         raise ValueError("Error:" + " response from server was: " + str(response.text))
 
-
 def humio_list_notifiers(client, args, headers):
-    data: Dict[str, str] = {}
-    url = "/api/v1/repositories/" + args.get("repository") + "/alertnotifiers"
+    url = "/graphql"
     headers["Accept"] = "application/json"
-    response = client.http_request("GET", url, data, headers)
+
+    graphql_query = """
+    query{searchDomain(name:"{repoName}"){actions{__typename,id, name 
+        ... on EmailAction{id, name, recipients, subjectTemplate, emailBodyTemplate: bodyTemplate, useProxy, attachCsv}
+        ... on SlackAction{url, fields{fieldName, value}, useProxy}
+        ... on SlackPostMessageAction{apiToken, channels, fields{fieldName, value}, useProxy}
+        ... on WebhookAction{method, url, webhookBodyTemplate: bodyTemplate, headers{header,value}, ignoreSSL, useProxy}
+        ... on OpsGenieAction{apiUrl, genieKey, useProxy}
+        ... on VictorOpsAction{messageType, notifyUrl, useProxy}
+        ... on PagerDutyAction{severity, routingKey, useProxy}
+        ... on HumioRepoAction{ingestToken}
+        ... on UploadFileAction{fileName}}}}
+    """.format(repoName=args.get("repository"))
+
+    data = {"query" : graphql_query}
+    json_data = json.dumps(data)
+
+    response = client.http_request("POST", url, json_data, headers).json()
+
     if response.status_code == 200:
-        result = response.json()
-        markdown = tableToMarkdown("Humio Notifiers", result, removeNull=True)
-        outputs = {"Humio.Notifier(val.id == obj.id)": result}
-        return markdown, outputs, result
+        if not response.get("data"):
```

Comment:
In this code fragment you are extracting the json response with a .json() This should return a dict object. In line 259  you are using response.status_doce did you check that this works? because I think this may lead to an Exception. 
please look at the code before this pr. first checking for the status_code and then accessing the response.json(). Also look at your other function that deal with responses they handle it the right way.

Reply:
@AlexanderBrandborg Do you have any thoughts on this? Looks like there's one too many `.json()`'s in there?